crates/c-api/CMakeLists.txt

```diff
@@ -78,6 +78,9 @@ if(ANDROID)
 endif()
 include(ExternalProject)
 find_program(WASMTIME_CARGO_BINARY cargo)
+if(NOT WASMTIME_CARGO_BINARY)
+    message(FATAL_ERROR [["cargo" was not found. Aborting...]])
```

Comment:
Might be worth also recommending a resolution such as adding cargo to PATH or to CMAKE_PROGRAM_PATH.